blog/docs/articles/getting-started-blog-p2.md

```diff
@@ -0,0 +1,275 @@
+# Getting Started in Open Source with Knative Part 2: Environment Setup 
+
+**Authors: [Calum Murray](https://www.linkedin.com/in/calum-ra-murray/) Software Engineering Intern @ [Red Hat](https://www.redhat.com/en), and [Leo Li](https://www.linkedin.com/in/haocheng-leo/) Software Engineering Intern @ [Red Hat](https://www.redhat.com/en)**
+
+**Date: 2023-07-26**
+
+Hi, and welcome back to this introductory blog series! In this article we will be discussing __how to set up your development environment__ for Knative.
+A development environment is a place where you can modify and examine code without affecting the currently released code. For setting up your Knative development
+environment, we are splitting the setup of the development environment into the local setup (i.e. things you set up on your own computer), and the external setup 
+(i.e. things you set up that aren't on your own computer).
```

Comment:
Would you like to give a link to the 1st article here?

And similarly, give a link to the this one at the end of the previous article?